Source/Foundation/bsfUtility/Debug/BsDebug.h

```diff
@@ -98,6 +105,8 @@ namespace bs
 	private:
 		UINT64 mLogHash = 0;
 		Log mLog;
+		/** @copydoc START_UP_DESC::logCallback */
```

Comment:
We shouldn't copy-doc from `START_UP_DESC`. Anything in `bsfUtility` shouldn't reference classes in higher 'layers' (i.e. `bsfEngine`).

Reply:
What would be the alternative, have the copydoc going the other way?

Reply:
You can't copy-doc private stuff. I'd not document the private field at all since its setter is documented anyway.

Reply:
I removed the documentation on it.